pom.xml

```diff
@@ -180,7 +200,7 @@ flexible messaging model and an intuitive client API.</description>
             <groupId>log4j</groupId>
           </exclusion>
           <exclusion>
-            <groupId>org.jboss.netty</groupId>
+            <groupId>io.netty</groupId>
```

Comment:
This one is actually the version of netty 3 used by ZK client

Reply:
3.4.6 is already on the io groupId

http://search.maven.org/#artifactdetails%7Corg.apache.zookeeper%7Czookeeper%7C3.4.6%7Cpom

Reply:
Got it, thanks!